src/cli/serve/__tests__/reload_logging_config.js

```diff
@@ -1,95 +0,0 @@
-import { spawn } from 'child_process';
-import { writeFileSync } from 'fs';
-import { relative, resolve } from 'path';
-import { safeDump } from 'js-yaml';
-import es from 'event-stream';
-import { readYamlConfig } from '../read_yaml_config';
-import expect from 'expect.js';
-
-const testConfigFile = follow(`fixtures/reload_logging_config/kibana.test.yml`);
-
-function follow(file) {
-  return relative(process.cwd(), resolve(__dirname, file));
-}
-
-function setLoggingJson(enabled) {
-  const conf = readYamlConfig(testConfigFile);
-  conf.logging = conf.logging || {};
-  conf.logging.json = enabled;
-  const yaml = safeDump(conf);
-  writeFileSync(testConfigFile, yaml);
-  return conf;
-}
-
-describe(`Server logging configuration`, function () {
```

Comment:
I ended up rewriting this test to make it a bit more clear. It's still not great, but it's better imo.